.github/workflows/codeql-analysis.yml

```diff
@@ -1,56 +0,0 @@
-name: 'CodeQL'
```

Comment:
@henningmu Did you mean to delete this?

Reply:
Yup, it didn't run for 7 months anyway, cleaning the house a little 🧹